src/proof/veriT/veriT_post_processor.h

```diff
@@ -0,0 +1,98 @@
+/*********************                                                        */
+/*! \file veriT_post_processor.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Hanna Lachnitt
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief The module for processing proof nodes into veriT proof nodes
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef CVC4__PROOF__VERIT_PROOF_PROCESSOR_H
+#define CVC4__PROOF__VERIT_PROOF_PROCESSOR_H
+
+#include <map>
+#include <unordered_set>
+
+#include "expr/proof_node_updater.h"
+#include "proof/veriT/veriT_proof_rule.h"
+
+namespace CVC4 {
+
+namespace proof {
+
+/**
+ * A callback class used by the veriT convereter for post-processing proof nodes
+ * by replacing internal rules by the rules in the veriT calculus.
+ */
+class VeriTProofPostprocessCallback : public ProofNodeUpdaterCallback
+{
+ public:
+  VeriTProofPostprocessCallback(ProofNodeManager* pnm);
+  ~VeriTProofPostprocessCallback() {}
+  /**
+   * Initialize, called once for each new ProofNode to process. This initializes
+   * static information to be used by successive calls to update.
+   */
+  void initializeUpdate();
+  bool shouldUpdate(std::shared_ptr<ProofNode> pn, 	
+		    bool& continueUpdate) override;
+  /** Update the proof rule application. */
+  bool update(Node res,
+              PfRule id,
+              const std::vector<Node>& children,
+              const std::vector<Node>& args,
+              CDProof* cdp,
+	      bool& continueUpdate) override;
```

Comment:
I see that you are using tabs. :) Clang-format should remove them (and I think that trailing spaces as well, at least in .cpp/.h files) but still, maybe better to avoid introducing them in the first place.